packages/gen-ai-hub/src/utils/deployment-resolver.ts

```diff
@@ -1,6 +1,8 @@
-import { DeploymentApi, AiDeployment } from '@sap-ai-sdk/ai-core';
+import { DeploymentApi, type AiDeployment } from '@sap-ai-sdk/ai-core';
 import { CustomRequestConfig } from '@sap-ai-sdk/core';
 import { pickValueIgnoreCase } from '@sap-cloud-sdk/util';
+import { deploymentIdCache } from './deployment-cache.js';
```

Comment:
[q] If this is shared code between the clients, I think it should not lie in the gen-ai-hub package? Maybe ai-core?

Reply:
good call, I just left it where the deployment resolver was. I would like to do this on a separate PR, as this would likely lead to issues with my other PR (deployment resolver).